src/GraphQL.Tests/Federation/FederationResolverTests.cs

```diff
@@ -467,33 +466,13 @@ private static T TestResolver<T>(IObjectGraphType objectGraphType, string repres
 
     private class MySchemaTypes : SchemaTypes
     {
-        private readonly Dictionary<ROM, IGraphType> _types = [];
-
         public MySchemaTypes(IEnumerable<IGraphType> types)
         {
             foreach (var type in types)
             {
-                _types.Add(type.Name, type);
+                Dictionary.Add(type.Name, type);
             }
         }
-
-        protected internal override Dictionary<ROM, IGraphType> Dictionary => _types;
-    }
-
-    private IFederationResolver GetResolver(IGraphType objectGraphType)
-    {
-        return GetResolvers(objectGraphType).ShouldHaveSingleItem();
-    }
-
-    private IEnumerable<IFederationResolver> GetResolvers(IGraphType objectGraphType)
```

Comment:
`GetResolver` and `GetResolvers` are not referenced in the codebase, so I removed them.